subiquity/client/controllers/ubuntu_advantage.py

```diff
@@ -33,13 +44,18 @@ class UbuntuAdvantageController(SubiquityTuiController):
 
     endpoint_name = "ubuntu_advantage"
 
+    def __init__(self, app):
+        """ Initializer for client-side UA controller. """
+        strategy: UAInterfaceStrategy
+        if app.opts.dry_run:
+            strategy = MockedUAInterfaceStrategy()
+        else:
+            strategy = UAClientUAInterfaceStrategy()
+        self.ua_interface = UAInterface(strategy)
+        super().__init__(app)
+
     async def make_ui(self) -> UbuntuAdvantageView:
         """ Generate the UI, based on the data provided by the model. """
-
-        # TODO remove these two lines to enable this screen
-        await self.endpoint.skip.POST()
-        raise Skip("Hiding the screen until we can validate the token.")
```

Comment:
I think this means that on a non-LTS, the user would see
1. screen is presented
2. user inputs 30 character token
3. subiquity informs the user that the release is not supported

Is that accurate?